src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1542,6 +1542,9 @@ public async Task<JObject> InvokeMethod(ArraySegment<byte> valueTypeBuffer, int
 
         public async Task<JObject> InvokeMethodInObject(int objectId, int methodId, string varName, CancellationToken token)
         {
+            valueTypes.TryGetValue(objectId, out var valueType);
+            if (valueType != null)
```

Comment:
```csharp
if (valueTypes.TryGetValue(objectId, out var valueType))
    return await InvokeMethod(valueType.valueTypeBuffer, methodId, varName, token);
```

`Try*` pattern returns `true` when the call succeeds, and the `out` param has a valid value.